### ISSUE TITLE
Is geddy project is dead?

### ISSUE THREAD

**Issue by maqboolkhan** [geddy/geddy#715]
I want to know is geddy is dead? because there no activity on github since last  year ??

**Comment by mde** [comment]
I would not say Geddy is dead. Geddy lives in all our hearts. :) Seriously though, it's pretty much in maintenance mode, unless people want to contribute. I am much too busy with my current job to spend much time on it. Thank you for asking though!

**Comment by mde** [comment]
I have added you to the Geddy org, so you can make changes and contributions if you'd like.

**Comment by gabe-alex** [comment]
@mde 
Umm... about that, I wanted to say this when you added me but, don't you you think that it may be a bad idea to just add any random people to the organisation?
I think you should add only those who specifically ask for it. It's pointless to add someone who's not interested, or doesn't have the time to contribute.
And there's also the problem of security, what if you add someone who wants to destroy the project for whatever reason? At least you should use a different group for new people, not the fucking Owner one with all privileges...

**Comment by mde** [comment]
This is a very good question, and it's encouraging to see that there are people who actually care.

- Security is a not really an issue -- only a couple of people including me have publish rights on NPM, and you shouldn't be running a master branch in prod, period. Nobody has access to the documentation site except for me at this point.
- If the person isn't interested, they simply don't respond to the invitation. (I am assuming here if the person is asking about the status of the project, they might actually care.)

I suppose you might be right about the Owner group, although people coming in asking if the project is dead would seem highly unlikely to come in then and kill it. At this point, it seems much more likely that it will die a natural death than that someone will kill it.

I tend to want to trust people, but if you'd like to set up some more granular ownership and permissions, let's do that, by all means.

**Comment by gabe-alex** [comment]
You have some valid points for this user's particular case.
But as a tip, just because a user's interested in the project's state, doesn't mean they want to develop for it. Most people will just ask that from a user perspective, to determine if it's a good idea to use geddy or not.

**Comment by mde** [comment]
You're absolutely right -- of course, if the person isn't interested, they can simply ignore the GitHub invite to the org. In any case, would be great to get some more participation. I guess if we're having this discussion, it's not dead!

**Comment by maqboolkhan** [comment]
thank you @mde  i am really thankfull for your github contributor invitation first i will check my time then i will accept invitation otherwise not and i dont worry if i accept i will not delete your repo and any thing stupid :smile:
